src/Surfnet/StepupMiddleware/CommandHandlingBundle/Configuration/Command/CreateInstitutionConfigurationCommand.php

```diff
@@ -19,8 +19,9 @@
 namespace Surfnet\StepupMiddleware\CommandHandlingBundle\Configuration\Command;
 
 use Surfnet\StepupMiddleware\CommandHandlingBundle\Command\AbstractCommand;
+use Surfnet\StepupMiddleware\CommandHandlingBundle\Command\ManagementExecutable;
 
-class CreateInstitutionConfigurationCommand extends AbstractCommand
+class CreateInstitutionConfigurationCommand extends AbstractCommand implements ManagementExecutable
```

Comment:
Think theory vs practice: in reality, which user of the middleware will be the instigator of the creation of an InstitutionConfiguration, even if the creation is derived from other intent? The command should be executable on behalf of those users, to give some limitation.